lib/Bio/EnsEMBL/DataCheck/Checks/CheckGenomicAlignTreeTable.pm

```diff
@@ -105,6 +105,30 @@ sub tests {
 
   }
 
+  my $sql_4 = qq/
+    SELECT COUNT(*)
+      FROM genomic_align_tree gat
+        JOIN genomic_align ga USING (node_id)
+        JOIN dnafrag df USING (dnafrag_id)
+      WHERE gat.right_index-gat.left_index!=1
+        AND df.coord_system_name!='ancestralsegment'
+    /;
+
+  my $desc_4 = "All internal nodes are ancestral sequences";
+  is_rows_zero($dbc, $sql_4, $desc_4);
+
+  my $sql_5 = qq/
+    SELECT COUNT(*)
+      FROM genomic_align_tree gat
+        JOIN genomic_align ga USING (node_id)
+        JOIN dnafrag df USING (dnafrag_id)
+      WHERE gat.right_index-gat.left_index=1
+        AND df.coord_system_name='ancestralsegment';
```

Comment:
```suggestion
        AND df.coord_system_name = 'ancestralsegment'
```
(not necessary, just removing ';' for consistency, the spaces included for eye breathers)